modules/i18n/src/translations/en-US/portals/policy-administration.ts

```diff
@@ -85,6 +85,32 @@ export const policyAdministration: policyAdministrationNS = {
     },
     policyAlgorithm: {
         actionText: "Select a policy combining algorithm",
+        algorithmOptions: {
+            denyOverrides: {
+                description: "The deny overrides combining algorithm is intended for those cases where a deny decision should have priority over a permit decision."
+            },
+            denyUnlessPermit: {
+                description: "This algorithm denies access unless explicitly permitted. If none of the policies produce a \"Permit,\" the result is \"Deny.\" However, if any policy results in a \"Permit,\" the final decision will be \"Permit.\""
```

Comment:
Denies unless explicitly permitted. If no policy results in a "Permit", the final decision is "Deny".